.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+on: [push]
+
+name: Go
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: ["1.12.8"]
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
```

Comment:
Behind this innocuous looking line lies dragons:

https://github.com/actions/setup-go/blob/master/src/installer.ts

What was wrong with `wget`?

Reply:
Travis had gimme, which was equally a big hack in bash.

Ultimately you can't just rely on wget, because that won't easily support many platforms, and "latest" versions like `1.12.x`.

Honestly, I don't care what setup-go does, as long as it works :)